Moved to using nullable references (#3068)
Revolutionary-Games/Thrive: doc/style_guide.md

  reading the code that they couldn't see directly from the method
  signature.

- For the main Thrive.csproj 
  [nullable reference types](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references) 

>   feature is turned on. All nullable reference warnings need to be 

Reviewer: ```suggestion
  are enabled. All nullable reference warnings need to be 
```

Author: Any other thoughts on this from someone else? I don't think strictly speaking what I wrote is incorrect grammatically or conceptually.

Reviewer 2: "is turned on" is more colloquial, but nobody will misunderstand it. I'd lean slightly towards "are enabled", but it's small potatos.
